### DISCUSSION TITLE
LAB 05.02

### DISCUSSION THREAD

**Discussion by joanfco30** [rramosp/ai4eng.v1#128] [05 - GESTIÓN DE MODELOS]
Buenas tardes un cordial saludo. Tengo una inquietud con respecto a esta tareas.  El termino non-random partitions lo estoy tomando como un array ordenado de acuerdo a tamaño de 'X' y 'y'. 
Considero que el proceso esta bueno,  pero al calificar esta tarea, estoy seguro que mi error esta en esta linea de código n_r_partition = np.arange(0, (X.shape[0])). De lo contrario, es posible una pequeña guía para continuar desarrollando esta tareas. 

Muchas gracias

![image](https://user-images.githubusercontent.com/60493083/169717797-290e5d26-bff7-429a-80fb-5a3360eb981e.png)

**Comment by joanfco30** [comment]
Buenos días, ya note mi error. Muchas gracias

**Reply by Felipe-RA** [reply]
Gracias por compartir joan
